### PR TITLE
RT-5.6 Interface Loopback mode

### PR DESCRIPTION
**Pull request by cprabha**
Hi,

Please find automation for RT-5.6 Interface Loopback mode.

Resolves: https://github.com/openconfig/featureprofiles/issues/1449

Thanks,
Prabha

### CODE REVIEW

**Review by sachendras — DISMISSED** [review]

File: feature/experimental/interface/loopback/ate_tests/loopback_aggregate_test/interface_loopback_aggregate_test.go
Comment (greg-dennis) [inline feature/experimental/interface/loopback/ate_tests/loopback_aggregate_test/interface_loopback_aggregate_test.go:151]: NextBundleInterface is deprecated. Please use NextAggregateInterface instead.

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: feature/experimental/interface/loopback/ate_tests/loopback_aggregate_test/interface_loopback_aggregate_test.go
Comment (cprabha) [reply feature/experimental/interface/loopback/ate_tests/loopback_aggregate_test/interface_loopback_aggregate_test.go]: updated, thank you

**Review by greg-dennis — APPROVED** [review]
Approving Go quality

**Review by greg-dennis — APPROVED** [review]

**Review by greg-dennis — CHANGES_REQUESTED** [review]
We have an internal issue blocking the merge of this, I am still investigating whether we can unblock this internally.

**Comment by cprabha** [comment]
> We have an internal issue blocking the merge of this, I am still investigating whether we can unblock this internally.

@greg-dennis , This Pull request shows "Requested changes must be addressed to merge this pull request.
" , I have updated all the changes requested.   I could not change the status on this PR. please could you check.

**Review by greg-dennis — APPROVED** [review]

**Review by sachendras — APPROVED** [review]

**Comment by cprabha** [comment]
@greg-dennis , @sachendras I have updated PR with OTG files. please check. Thanks

**Review by sachendras — APPROVED** [review]

File: feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go
Comment (cprabha) [reply feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go:218]: yes, we are enabling interface loopback mode and that will bring up AE and intf .

File: feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go
Comment (cprabha) [reply feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go:201]: Lower layer down is right one here since underlying intf is down.

File: feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go
Comment (cprabha) [reply feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go:215]: not sure of the version check, teststeps does not specify anything about version check.

File: feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go
Comment (manan-patel) [reply feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go:201]: The request is to uplift the tc with OR condition like the other tc

File: feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go
Comment (manan-patel) [reply feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go:215]: It's a question for the infra team, so do we have it supported?

**Pull request merged**
